lib/Optimizer/Transforms/RegToMem.cpp

```diff
@@ -143,14 +143,19 @@ struct RegToMemAnalysis {
       }
     });
     func.walk([&](Operation *op) {
+      // FIXME: This really needs to figure out which operands are wires and
+      // which are not. It blindly assumes everything is a wire.
```

Comment:
Should this be captured in a new issue?

Reply:
Yes.

Reply:
https://github.com/NVIDIA/cuda-quantum/issues/763